pages/events/event_view.php

```diff
@@ -52,15 +53,28 @@
             </div>
         </div>
 
+        <?php if ($has_file): ?>
+            <div class="file-button">
+                <a href="/download?id=<?= $event["circu"] ?>" role="button">
+                    <div>
+                        <i class="fas fa-paperclip"></i>
+                    </div>
+                    <div>
+                        <b>Informations</b>
```

Comment:
I would have maybe displayed a bit more info here, maybe the file name? I would say that changing the structure of the table also makes sense, and maybe hashing the name of the files. However we can do that in a later PR, this is good enough for now i'd say :)